ansible/roles/jicofo/defaults/main.yml

```
@@ -1,6 +1,8 @@
 ---
-jicofo_auth_domain: auth.domain.local
-jicofo_auth_password: replaceme
+jicofo_auth_domain: "auth.{{ prosody_domain_name | default(domain.local) }}"
```

```suggestion
jicofo_auth_domain: "auth.{{ prosody_domain_name | default('domain.local') }}"
```
default values need to be python values, so quotes around strings